packages/react-native-jigsaw/src/components/Button.tsx

```diff
@@ -146,15 +146,38 @@ const Button: React.FC<Props> = ({
     },
   ];
 
+  const {
+    margin,
+    marginEnd,
+    marginTop,
+    marginLeft,
+    marginRight,
+    marginBottom,
+    marginHorizontal,
+    marginVertical,
```

Comment:
I don't even think we expose these in the builder, but I figured better safe than sorry....